ui/tests/subscribe.test.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect } from '@Test';
```

Comment:
Do you have conflicts in 2 tests - ui/tests/subscribe.test.ts & ui/tests/wish.test.ts